Resources/Prototypes/Catalog/Fills/Boxes/general.yml

```diff
@@ -1,6 +1,21 @@
 - type: entity
-  name: lightbulb box
+  name: cardboard box
   parent: BoxBase
+  id: BoxCardboard
+  description: A cardboard box for storing things.
+  components:
+  - type: Sprite
+    layers:
+      - state: box
+  - type: Item
+    size: 20
+  - type: Storage
+    capacity: 20
+    size: 20
```

Comment:
Is this sprite netsynced?

Reply:
it's inheriting from the boxBase so it shouldn't be

![image](https://user-images.githubusercontent.com/47410468/147414420-3b22661c-9645-4f75-ad8f-30f6a083a24c.png)